script.js

```diff
@@ -198,15 +203,24 @@ exit.addEventListener('click', () => {
   contact.classList.remove('blur');
   brand.classList.remove('blur');
 });
+let bool;
 
-function fix() {
-  const mail = document.forms['contact-form'].email.value;
-  const error = document.getElementById('mail-error');
-  if (mail.toLowerCase() !== mail) {
+getinButton.addEventListener('click', () => {
+  bool = false;
+  const mail = emailInput.value;
+  if (mail.toLowerCase() !== mail && bool === false) {
     error.hidden = false;
-    return false;
+  } else if (mail.toLowerCase() === mail) {
+    localStorage.setItem('form', JSON.stringify({
+      name: nameInput.value,
+      email: emailInput.value,
+      textarea: textArea.value,
+    }));
+    bool = true;
   }
+});
 
-  return true;
-}
-fix();
+const oldform = JSON.parse(localStorage.getItem('form'));
+nameInput.value = oldform.name;
+emailInput.value = oldform.email;
+textArea.value = oldform.textarea;
```

Comment:
- Good job so far on correctly using the browser's local storage method. :+1: , but I think there are some changes you need to make before moving to the next milestone:
- [ ] When a user loads the browsers, before getting the form properties, it's a good practice to first check if the item exists on the local storage before accessing its properties, please try to wrap this in an if statement.
- If you implement this correctly, this error wouldn't display in the console.
![Screenshot from 2023-04-13 18-44-25](https://user-images.githubusercontent.com/107632780/231842578-a8da5400-848e-4af6-85cb-437e842555e4.png)